util/cron/common-whitebox.bash

```diff
@@ -94,7 +94,7 @@ if [ "${COMPILER}" != "gnu" ] ; then
     # module load gcc/${CHPL_GCC_TARGET_VERSION}
     # For now, we need to force it to gcc 6.3.0 so its libraries will
```

Comment:
Need to update comment

Reply:
Sigh. This is what I get for rushing.

Reply:
Actually, so the reason it's explicitly setting it in this file is because it wants to override what's in compiler_versions.bash. If we're setting it to 7.3.0 in compiler_versions.bash as well, do we actually need this section of code? Or does the whitebox not pick up the x86_64 architecture section?

Reply:
Right, 6.3.0 should change to 7.3.0 in the comment as well.
The rest of the comment should stay; I really hope this workaround is still temporary.  In an ideal world, we would just load the gcc target version.

Reply:
What we're setting in compiler_versions.bash is 8.2.0 because this is the target compiler, not the gen compiler, so we still need to override.  (Sorry, I didn't see your message until after I posted mine.)

Reply:
Ah, got it.